web/src/components/pages/partner/partner-option-section.css

```diff
@@ -0,0 +1,35 @@
+.partner-options-container {
+    @media (--lg-up) {
+        display: flex;
+        justify-content: space-evenly;
+    }
+
+    margin-bottom: 32px;
+}
+
+.partner-options-container img {
+    align-self: end;
+}
+
+.partner-options-text-container {
+    flex: 0.8;
+}
+
+.partner-options-text-container h3 {
```

Comment:
Are there any other font classes (ideally generic) that look have similar css? I'd be good to ensure all our fonts use the same or similar css or make it so this style can be used more generally

Reply:
I'm not sure. I've looked at various pages in the application and I can't see any text that looks similar to this